### PR TITLE
[EKA-35] Introducing callBackUrl

### PR DESCRIPTION
**Pull request by Apoorva-GA**

### CODE REVIEW

File: src/main/java/in/org/projecteka/hiu/consent/model/Consent.java [hunk 1]
```diff
@@ -15,14 +15,15 @@
...
-import static in.org.projecteka.hiu.consent.model.consentmanager.Frequency.ZERO_HOUR;
+import static in.org.projecteka.hiu.consent.model.consentmanager.Frequency.ONE_HOUR;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 @AllArgsConstructor
 @Builder
 @NoArgsConstructor
 @Data
 public class Consent {
+    private static final String CALL_BACK_URL = "http://hiu-dev.projecteka.in/";
```
Comment (angshu) [inline src/main/java/in/org/projecteka/hiu/consent/model/Consent.java]: This should not be hardcoded. at the least, it should be an application property. I could possibly think of the callback URL to be dynamic, but probably later

File: src/main/java/in/org/projecteka/hiu/consent/model/Consent.java [hunk 2]
```diff
@@ -15,14 +15,15 @@
 import java.util.List;
 import java.util.TimeZone;
 
-import static in.org.projecteka.hiu.consent.model.consentmanager.Frequency.ZERO_HOUR;
+import static in.org.projecteka.hiu.consent.model.consentmanager.Frequency.ONE_HOUR;
```
Comment (angshu) [inline src/main/java/in/org/projecteka/hiu/consent/model/Consent.java:18]: best keep these as application properties

File: src/main/java/in/org/projecteka/hiu/consent/model/consentmanager/Frequency.java [hunk 3]
```diff
@@ -12,5 +12,5 @@
 public class Frequency {
     private Unit unit;
     private int value;
-    public static Frequency ZERO_HOUR = new Frequency(Unit.HOUR, 0);
+    public static Frequency ONE_HOUR = new Frequency(Unit.HOUR, 1);
```
Comment (angshu) [inline src/main/java/in/org/projecteka/hiu/consent/model/consentmanager/Frequency.java:15]: best not define as constants (unless meant as default value).  in that case, "Unit.HOUR" can be read from application properties.

**Review by angshu — CHANGES_REQUESTED** [review]

File: src/main/java/in/org/projecteka/hiu/consent/model/consentmanager/Frequency.java (hunk 3 above)
Comment (Apoorva-GA) [reply src/main/java/in/org/projecteka/hiu/consent/model/consentmanager/Frequency.java:15]: These are default values for now, hence defined as above.

**Review by angshu — APPROVED** [review]

**Pull request merged**